src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
+
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__item {
+    margin-left: 20px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $headerHigh;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition-delay: 0.2s;
```

Comment:
`transition-delay` doesn't work in this case, use `transition` instead

Reply:
Fixed